src/precision.hpp

```diff
@@ -107,6 +107,49 @@ inline static Precision fromString(const std::string& s) {
     return it->second;
 }
 
+inline static Precision kfsPrecisionToOvmsPrecision(const std::string& s) {
+    static std::unordered_map<std::string, Precision> precisionMap{
+        {"BOOL", Precision::BOOL},
+        {"FP64", Precision::FP64},
+        {"FP32", Precision::FP32},
+        {"FP16", Precision::FP16},
+        {"INT64", Precision::I64},
+        {"INT32", Precision::I32},
+        {"INT16", Precision::I16},
+        {"INT8", Precision::I8},
+        {"UINT64", Precision::U64},
+        {"UINT32", Precision::U32},
+        {"UINT16", Precision::U16},
+        {"UINT8", Precision::U8}};
+    auto it = precisionMap.find(s);
+    if (it == precisionMap.end()) {
+        return Precision::UNDEFINED;
+    }
+    return it->second;
+}
+
+inline static std::string ovmsPrecisionToKfsPrecision(Precision precision) {
+    static std::unordered_map<Precision, const char*> precisionMap{
+        {Precision::FP64, "FP64"},
+        {Precision::FP32, "FP32"},
+        {Precision::FP16, "FP16"},
+        {Precision::I64, "INT64"},
+        {Precision::I32, "INT32"},
+        {Precision::I16, "INT16"},
+        {Precision::I8, "INT8"},
+        {Precision::U64, "UINT64"},
+        {Precision::U32, "UINT32"},
+        {Precision::U16, "UINT16"},
+        {Precision::U8, "UINT8"},
+        {Precision::BOOL, "BOOL"},
```

Comment:
It's worth to list here known currently unsupported datatypes with TODO's

Reply:
done